### PR TITLE
Update 0000-01-17-c1kmd01.md

### PR DESCRIPTION
**Pull request by c1kmd01**
I am creating a pull request

### CODE REVIEW

File: _posts/0000-01-17-c1kmd01.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Ha Ha I changed the title"
```
Comment (SLEEP1NG) [inline _posts/0000-01-17-c1kmd01.md:3]: @c1kmd01 Nicely done.

**Review by c1kmd01 — COMMENTED** [review]
I updated this

File: _posts/0000-01-17-c1kmd01.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
-Caption here
+I am changing this post!!!!!!
```
Comment (SLEEP1NG) [inline _posts/0000-01-17-c1kmd01.md]:
```suggestion
I am changing this post!!!!!! | You may want to caption the image you were assigned.
```

**Review by beardofedu — APPROVED** [review]

**Review by SLEEP1NG — CHANGES_REQUESTED** [review]
Please update the caption to reflect the ice cream eating cat.

**Review by SLEEP1NG — APPROVED** [review]

**Pull request merged**